account_invoice_reorder_lines/__openerp__.py

```diff
@@ -23,17 +23,17 @@
     'name': 'Invoice lines with sequence number',
     'version': '0.1',
```

Comment:
Please update it according to https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers